### PR TITLE
LRAC-11900 [F] Implement Empty State display message

### PR DESCRIPTION
**Pull request by fmarcio**
Jira issue https://issues.liferay.com/browse/LRAC-11900

### CODE REVIEW

File: modules/apps/analytics/analytics-settings-web/src/main/resources/META-INF/resources/js/pages/wizard/PropertyStep.tsx [hunk 1]
```diff
@@ -12,12 +12,69 @@
...
+const Step: React.FC<IStepProps> = ({onChangeStep}) => {
+	return (
+		<BasePage
+			description={Liferay.Language.get('property-description')}
+			title={Liferay.Language.get('property-assignment')}
+		>
+			<div className="text-right">
+				<ClayButton
+					displayType="secondary"
+					onClick={() => alert('cria nova property')}
```
Comment (fmarcio) [inline modules/apps/analytics/analytics-settings-web/src/main/resources/META-INF/resources/js/pages/wizard/PropertyStep.tsx:33]: TODO: replace this with modal

File: modules/apps/analytics/analytics-settings-web/src/main/resources/META-INF/resources/js/pages/wizard/PropertyStep.tsx [hunk 2]
```diff
@@ -12,12 +12,69 @@
...
+					imgProps={{
+						alt: 'Create a New Property',
+						title: 'Create a New Property',
+					}}
+					imgSrc={`${Liferay.ThemeDisplay.getPathThemeImages()}/states/search_state.gif`}
+					title={Liferay.Language.get('create-a-new-property')}
+				>
+					<ClayButton
+						displayType="secondary"
+						onClick={() => alert('cria nova property')}
```
Comment (fmarcio) [inline modules/apps/analytics/analytics-settings-web/src/main/resources/META-INF/resources/js/pages/wizard/PropertyStep.tsx:54]: TODO: replace this with modal

**Comment by fmarcio** [comment]
ci:test:analytics-cloud-acceptance

**Comment by fmarcio** [comment]
ci:test:relevant

**Comment by fmarcio** [comment]
ci:test:sf

**Comment by fmarcio** [comment]
ci:test:stable

**Comment by fmarcio** [comment]
ci:test:master:analytics-cloud-acceptance

File: modules/apps/analytics/analytics-settings-web/src/main/resources/META-INF/resources/js/pages/wizard/PropertyStep.tsx [hunk 3]
```diff
@@ -12,12 +12,75 @@
...
+			description={Liferay.Language.get('property-description')}
+			title={Liferay.Language.get('property-assignment')}
+		>
+			<div className="text-right">
+				<ClayButton
+					displayType="secondary"
+
+					// TODO: Replace this alert with modal of LRAC-11980
+
+					onClick={() => alert('cria nova property')}
```
Comment (interaminense) [inline modules/apps/analytics/analytics-settings-web/src/main/resources/META-INF/resources/js/pages/wizard/PropertyStep.tsx:36]: Could you just return a empty fn?
```suggestion
					onClick={() => {}}
```

File: modules/apps/analytics/analytics-settings-web/src/main/resources/META-INF/resources/js/pages/wizard/PropertyStep.tsx [hunk 4]
```diff
@@ -12,12 +12,75 @@
...
+					}}
+					imgSrc={`${Liferay.ThemeDisplay.getPathThemeImages()}/states/search_state.gif`}
+					title={Liferay.Language.get('create-a-new-property')}
+				>
+					<ClayButton
+						displayType="secondary"
+
+						// TODO: Replace this alert with modal of LRAC-11980
+
+						onClick={() => alert('cria nova property')}
```
Comment (interaminense) [inline modules/apps/analytics/analytics-settings-web/src/main/resources/META-INF/resources/js/pages/wizard/PropertyStep.tsx:60]:
```suggestion
						onClick={() => {}}
```

File: modules/apps/analytics/analytics-settings-web/src/main/resources/META-INF/resources/js/pages/wizard/PropertyStep.tsx [hunk 5]
```diff
@@ -12,12 +12,75 @@
...
+			</div>
+
+			<div className="empty-state-border">
+				<ClayEmptyState
+					description={Liferay.Language.get(
+						'create-a-property-to-add-sites-and-channels'
+					)}
+					imgProps={{
+						alt: 'Create a New Property',
+						title: 'Create a New Property',
```
Comment (interaminense) [inline modules/apps/analytics/analytics-settings-web/src/main/resources/META-INF/resources/js/pages/wizard/PropertyStep.tsx:50]: Please, use language key here

**Review by interaminense — CHANGES_REQUESTED** [review]
Hey @fmarcio, Just few comments on your PR

File: modules/apps/analytics/analytics-settings-web/src/main/resources/META-INF/resources/js/pages/wizard/PropertyStep.tsx (hunk 3 above)
Comment (fmarcio) [reply modules/apps/analytics/analytics-settings-web/src/main/resources/META-INF/resources/js/pages/wizard/PropertyStep.tsx]: @interaminense done!

File: modules/apps/analytics/analytics-settings-web/src/main/resources/META-INF/resources/js/pages/wizard/PropertyStep.tsx (hunk 4 above)
Comment (fmarcio) [reply modules/apps/analytics/analytics-settings-web/src/main/resources/META-INF/resources/js/pages/wizard/PropertyStep.tsx]: @interaminense done!

File: modules/apps/analytics/analytics-settings-web/src/main/resources/META-INF/resources/js/pages/wizard/PropertyStep.tsx (hunk 5 above)
Comment (fmarcio) [reply modules/apps/analytics/analytics-settings-web/src/main/resources/META-INF/resources/js/pages/wizard/PropertyStep.tsx]: @interaminense done!

**Review by ilzamcmed — APPROVED** [review]
Nice job @fmarcio 
LGTM! :rocket:

**Review by interaminense — APPROVED** [review]
LGTM! Thanks @fmarcio !

**Comment by interaminense** [comment]
ci:forward

**Comment by fmarcio** [comment]
SF test is failing because some files are missing. I'll have to generate them and add it on this PR

**Comment by fmarcio** [comment]
ci:test:sf

**Comment by fmarcio** [comment]
@interaminense FYI some other files were modified when I ran `yarn run liferay-npm-scripts types`

Please check them on this commit https://github.com/liferay-ac/liferay-portal/pull/120/commits/f8e0507bfe1decc69de6def9d2fd5567f094f779

**Comment by fmarcio** [comment]
ci:test:sf

**Comment by fmarcio** [comment]
ci:stop

**Comment by fmarcio** [comment]
Resent to https://github.com/liferay-ac/liferay-portal/pull/125

**Pull request closed without merging**